application/modules/filter/controllers/AjaxGuest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+if (!defined('BASEPATH')) exit('No direct script access allowed');
+
+/*
+ * InvoicePlane
+ *
+ * @author		InvoicePlane Developers & Contributors
+ * @copyright	Copyright (c) 2012 - 2017 InvoicePlane.com
+ * @license		https://invoiceplane.com/license.txt
+ * @link		https://invoiceplane.com
+ */
+
+/**
+ * Class Ajax
+ */
+class AjaxGuest extends Guest_Controller
```

Comment:
Please standardize class names to `Ajax_Guest` and alter all links to hyphen-separated lowercase links.

Reply:
ready